doc/man7/provider-base.pod

```diff
@@ -77,7 +89,7 @@ B<OSSL_get_{name}>.
 For example, the "function" core_gettable_params() has these:
 
  typedef OSSL_ITEM *
-     (OSSL_core_gettable_params_fn)(const OSSL_PROVIDER *prov);
+     (OSSL_core_gettable_params_fn)(const OSSL_CORE_HANDLE *handle);
```

Comment:
This isn't quite right any more...  the returned type is `const OSSL_PARAM *`.  Probably an omission on my part, but would you mind changing that while you're at it?

Reply:
Done